app/models/internal/impl/DefaultReport.java

```diff
@@ -60,8 +62,8 @@ public UUID getId() {
     }
 
     @Override
-    public String getETag() {
-        return _eTag;
+    public Optional<String> getETag() {
+        return Optional.ofNullable(_eTag);
```

Comment:
This should be an `Optional<...>` on the member by convention instead of wrapping on access. I realize this is against IDEA advice.

Reply:
I'm all for consistency even if it goes against convention. I'd rather that happen then have a ton of differing styles.